src/sentry/search/snuba/executors.py

```diff
@@ -325,9 +326,14 @@ def query(
         # to something that we can send down to Snuba in a `group_id IN (...)`
         # clause.
         max_candidates = options.get("snuba.search.max-pre-snuba-candidates")
-        too_many_candidates = False
-        group_ids = list(group_queryset.values_list("id", flat=True)[: max_candidates + 1])
+
+        with sentry_sdk.start_span(op="snuba_group_query") as span:
```

Comment:
The `op` name could be changed. I'm mildly concerned that the relationship to the existing `start_snuba_query` span might be a little unclear.

To explain, this "Snuba group query" is not the Snuba query proper, but a query that the Snuba executor is making against the Postgres DB in order to set up the actual search query.